app/collins/controllers/actions/ipaddress/GetPoolsAction.scala

```diff
@@ -38,14 +39,25 @@ case class GetPoolsAction(
           } else {
             IpAddresses.getPoolsInUse()
           }
-        format(pools)
+        val ipmiPools: Set[String] =
+          if (all) {
+            IpmiInfo.AddressConfig.map(_.poolNames).getOrElse(Set())
+          } else {
+            // TODO: Change this to only return active pools?
+            IpmiInfo.AddressConfig.map(_.poolNames).getOrElse(Set())
```

Comment:
I prefer reading the configured pools from the config for IPMI tbh. However, we would need to take into account the fact that IPMI addresses can be "loose", and not adherent to any pool in the config.

I am open to either, however an evolution will definitely complicate things. I also think if we track pool with an ipmi_info entry, we will need to update it when people hit /api/asset/:tag/ipmi to create a new address or update existing...